fabric-structure-api-v1/build.gradle

```diff
@@ -0,0 +1,6 @@
+archivesBaseName = "fabric-structure-api-v1"
+version = getSubprojectVersion(project, "1.0.0")
+
+dependencies {
+	compile(project(path: ':fabric-api-base', configuration: 'dev'))
```

Comment:
Hmm, what does this module use from base api? If it doesn't use anything from it, you can omit this dependency.